### ISSUE TITLE
A bug in function ov.space.svg in OV 1.6.7

### ISSUE THREAD

**Issue by Zstark11** [Starlitnightly/omicverse#173]
I saw someone mention this ISSUE before, when it was OV 1.6.5 version, but I still encountered the problem in version 1.6.7

Run the code as：
```
import omicverse as ov
#print(f"omicverse version: {ov.__version__}")
import scanpy as sc
print(f"scanpy version: {sc.__version__}")
ov.utils.ov_plot_set()
adata = sc.read_visium(path=r'/mnt/d/单细胞数据/test/KJ', count_file='151676_filtered_feature_bc_matrix.h5')
adata.var_names_make_unique()

'''我们在omicverse `1.6.0`以上的版本中引入了空间特殊svg计算模块prost来代替scanpy的HVG，
如果你想使用scanpy的HVG可以在`ov.space.svg`中设置mode=`scanpy`或者使用下面的代码:
#adata=ov.pp.preprocess(adata,mode='shiftlog|pearson',n_HVGs=3000,target_sum=1e4)
#adata.raw = adata
#adata = adata[:, adata.var.highly_variable_features]'''
#预处理数据
sc.pp.calculate_qc_metrics(adata, inplace=True)
adata = adata[:,adata.var['total_counts']>100]
adata=ov.space.svg(adata,mode='prost',n_svgs=3000,target_sum=1e4,platform="visium",)

```
ERROR
```
numba.core.errors.TypingError: Failed in nopython mode pipeline (step: nopython frontend)
Untyped global name 'tqdm': Cannot determine Numba type of <class 'type'>
File "../../../../../../home/zouguoyin/anaconda3/lib/python3.12/site-packages/omicverse/externel/PROST/utils.py", line 102:
def get_image_idx_1D(image_idx_2d):
    <source elided>
    image_idx_1d = np.ones(max_value).astype(int)
    for i in tqdm(range(1, max_value + 1)):
    ^
```
Numba version：0.6.0

**Comment by Starlitnightly** [comment]
I can't reproduce your reported error. Maybe you need to provide a more detailed version of the error.

```shell
#Perhape this version maybe helpful
numpy==1.23.5 llvmlite==0.41.1 numba==0.58.1
```

**Comment by Zstark11** [comment]
The problem was fixed when I lowered the Python version to 3.10 and changed the version of the library below to numpy==1.23.5 llvmlite==0.41.1 numba==0.58.1
